covariance/covarianceXsec.cpp

```diff
@@ -414,8 +419,39 @@ void covarianceXsec::PrintNormParams() {
     if (NormParams[i].pdgs.empty()) pdgString += "all";
 
     MACH3LOG_INFO("│{: <4}│{: <10}│{: <40}│{: <20}│{: <20}│{: <20}│", i, NormParams[i].index, NormParams[i].name, intModeString, targetString, pdgString);
+
+    if(NormParams[i].hasKinBounds) have_parameter_with_kin_bounds = true;
   }
   MACH3LOG_INFO("└────┴──────────┴────────────────────────────────────────┴────────────────────┴────────────────────┴────────────────────┘");
+
+  if(have_parameter_with_kin_bounds) {
+    MACH3LOG_INFO("Normalisation parameters KinematicCuts information");
+    MACH3LOG_INFO("┌────┬──────────┬────────────────────────────────────────┬────────────────────┬────────────────────────────────────────┐");
+    MACH3LOG_INFO("│{0:4}│{1:10}│{2:40}│{3:20}│{4:40}│", "#", "Global #", "Name", "KinematicCut", "Value");
+    MACH3LOG_INFO("├────┼──────────┼────────────────────────────────────────┼────────────────────┼────────────────────────────────────────┤");
+    for (unsigned int i = 0; i < NormParams.size(); ++i)
+      {
+	//skip parameters with no KinematicCuts
+	if(NormParams[i].hasKinBounds) continue;
```

Comment:
`!` is missing
I mean
`if(NormParams[i].hasKinBounds) continue;`

Reply:
That teaches me not to change logic when I discover a new data member... sorry!